Mage/Archivist/Archivist.cs

```diff
@@ -546,7 +561,9 @@ public void MGet(JObject queries, JObject options, Action<Exception, JToken> cb)
 
 						// Add value to response
 						string responseKey = realQueryKeys[cacheKeyName];
-						responseObject.Add(responseKey, GetCacheValue(cacheKeyName).Data);
+						var cacheValue = GetCacheValue(cacheKeyName);
+
+						responseObject.Add(responseKey, (cacheValue != null) ? cacheValue.Data : null);
```

Comment:
again null check is not needed here since `ValueSetOrDelete` will ensure the cache value exists
this will potentially swallow real errors